versions.yaml

```diff
@@ -189,6 +189,16 @@ externals:
     # version older than them.
     version: "v1.5.2"
 
+  containerd:
```

Comment:
@lifupan Do we need the `cri-containerd` version on line 182. I reckon we can take that out as well as part of this PR.